### DISCUSSION TITLE
A Few Feature Ideas

### DISCUSSION THREAD

**Discussion by mbtechtt** [uxbox/uxbox#1542] [Ideas]
Thanks to the whole penpot team for all the hard work they are putting into this free open source project. As an aspiring web developer I use penpot to make designs for my web projects. Penpot being the only 'Design Software' I have used personally, it has been amazing. A Few Features I would love to see implemented are:

- Backgrounds or Rectangles that scale with text for Button Items (A sort of Padding System could work where you choose the distance you want on the left, right, top and bottom of the text or a group with the buttons icon and text) 
- This is an example of what happens currently when you change the text.
![image](https://user-images.githubusercontent.com/49920384/152535235-7bab37c1-298a-403e-b023-810af8bd2ab4.png)
![image](https://user-images.githubusercontent.com/49920384/152535397-0e129751-a8b8-4953-b85a-f0e7a3248759.png)

- Better Image Tools Like Cropping (I saw some cool stuff with masking in the recent penpot live stream but just croping to a specific part of the image without using an exclude mask would be awesome)
- Pen Tablet Support for the Curve and Path Tools would be amazing when making things like masks :)

Keep up the Great Work.

**Comment by hirunatan** [comment]
Thanks for your suggestions. 

The first one you talk about is included in a functionality named "auto layout", that is one of our prioritized US. You can check it in https://tree.taiga.io/project/penpot/us/2262

For the image cropping I have created an issue here: https://tree.taiga.io/project/penpot/issue/2757

And the tablet support is a thing on which we are currently working, it will be important enhancements in the next release, stay tuned!
